docs/providers/openwhisk/cli-reference/invoke-local.md

```diff
@@ -18,6 +18,8 @@ This runs your code locally by emulating the Apache OpenWhisk environment. Pleas
 serverless invoke local --function functionName
 ```
 
+__*Please note that only the JavaScript and Python runtimes are supported with this command.*__
+
```

Comment:
I can change this to be "Node.js" but I assume most people will understand what this refers too?